### PR TITLE
Clippy

### PR DESCRIPTION
**Pull request by dannywillems**

### CODE REVIEW

File: rust-toolchain [hunk 1]
```diff
@@ -0,0 +1 @@
+1.75
```
Comment (mimoo) [inline rust-toolchain:1]: I think a `rust-toolchain.toml` is more idiomatic now no?

**Review by mimoo — APPROVED** [review]
thanks so much man :)

**Comment by mimoo** [comment]
looks like a toolchain issue

File: rust-toolchain (hunk 1 above)
Comment (dannywillems) [reply rust-toolchain]: Updated.

**Comment by dannywillems** [comment]
> looks like a toolchain issue

updated

**Pull request merged**